src/components/charts/sizeDistancePlotter/index.jsx

```diff
@@ -382,7 +383,7 @@ class SizeDistancePlotter extends React.Component {
                   return (
                     <Points
                       key={setId}
-                      data={set}
+                      data={set.data || set}
```

Comment:
I think probably this or is superfluous at this time.  We only have one json file to send to this plot and we're in charge of how it's structured so, for the time being, it's always `set.data`, right?  Or am I thinking about this wrong?

Reply:
I think this check, in this case, is certainly not need. I'm sure we'll remember to use this very basic method in the future if this ever requires the change.